apps/console/src/features/applications/components/wizard/minimal-application-create-wizard.tsx

```diff
@@ -167,7 +167,7 @@ export const MinimalAppCreateWizard: FunctionComponent<MinimalApplicationCreateW
     const orgType: OrganizationType = useSelector((state: AppState) =>
         state?.organization?.organizationType);
     const isFAPIAppCreationEnabled: boolean = useSelector((state: AppState) =>
-        state.config.ui.features?.fapiApplicationCreation?.enabled);
+        !state.config.ui.features?.applications?.disabledFeatures?.includes("fapiApplicationCreation"));
```

Comment:
Shall we move this to a constant in `ApplicationManagmentConstants` class.

Ref: https://github.com/wso2/identity-apps/blob/c96eb8b986c93451340dc1f2f3bd86201f7f2c81/modules/common/src/constants/organization-constants.ts#L26

Reply:
fixed with [dc28d6c](https://github.com/wso2/identity-apps/pull/4795/commits/dc28d6ceb08fe37ed124f1753cb954a516cea6ab)